mobile/library/objective-c/EnvoyEngine.h

```diff
@@ -35,7 +35,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param networkMonitoringMode Configure how the engines observe network reachability.
  */
 - (instancetype)initWithRunningCallback:(nullable void (^)())onEngineRunning
-                                 logger:(nullable void (^)(NSString *))logger
+                                 logger:(nullable void (^)(int, NSString *))logger
```

Comment:
The `NSString*` represents the message. I think using a struct is probably overkill given that we only have 2 parameters. Plus, this structure is similar to the C++ function pointer definition. As for using `NSNumber`, I think using `NSInteger` is better given that `LogLevel` is an `Int`: https://github.com/envoyproxy/envoy/blob/bb80b323080ff2ae62b4c18513f2d5093fd23bec/mobile/library/swift/LogLevel.swift#L6
So, I updated the code to use `NSInteger` instead of `int`.